src/sentinel.c

```diff
@@ -2263,7 +2279,7 @@ void rewriteConfigSentinelOption(struct rewriteConfigState *state) {
     rewriteConfigMarkAsProcessed(state,"sentinel auth-user");
     rewriteConfigMarkAsProcessed(state,"sentinel config-epoch");
     rewriteConfigMarkAsProcessed(state,"sentinel leader-epoch");
-    rewriteConfigMarkAsProcessed(state,"sentinel known-replica");
+    rewriteConfigMarkAsProcessed(state,knownReplicaConfig);
```

Comment:
open question - what happens if we have both "known-replica" and "known-slave" in the file?

Reply:
I think the config parameters are pre-config by human, thus generally known-replica and known-slave should not exist together.